### PR TITLE
renamed test function to remove underscore

### LINKED ISSUE #2555: Rename test functions of synchronizer to remove the `_` in the name
# Rationale

Rename test functions of synchronizer to remove the `_` in the name

### PR DESCRIPTION
**Pull request by joanestebanr**
Closes #2555 
### What does this PR do?
Renaming test functions to remove underscore

### Reviewers

Main reviewers:
@ARR552

### CODE REVIEW

File: synchronizer/l1_sync_orchestration_test.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ type mocksOrgertration struct {
 	consumer *l1RollupConsumerInterfaceMock
 }
 
-func Test_A(t *testing.T) {
+func TestOrquestrationTodo(t *testing.T) {
```
Comment (ARR552) [inline synchronizer/l1_sync_orchestration_test.go:14]: Is there an issue to implement it?

File: synchronizer/l1_rollup_info_producer_test.go [hunk 2]
```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func Test_L1Get(t *testing.T) {
+func TestExploratoryL1Get(t *testing.T) {
```
Comment (ARR552) [inline synchronizer/l1_rollup_info_producer_test.go:14]: Why is it skipped?

File: synchronizer/l1_worker_test.go [hunk 3]
```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func Test_Exploratory(t *testing.T) {
+func TestExploratoryWorker(t *testing.T) {
 	t.Skip("no real test, just exploratory")
```
Comment (ARR552) [inline synchronizer/l1_worker_test.go:13]: Why is it skipped?

File: synchronizer/l1_sync_orchestration_test.go (hunk 1 above)
Comment (joanestebanr) [reply synchronizer/l1_sync_orchestration_test.go:14]: No, basically is not done yet. And this PR is just a renaming, I'll do later to add the missing unittest

File: synchronizer/l1_rollup_info_producer_test.go (hunk 2 above)
Comment (joanestebanr) [reply synchronizer/l1_rollup_info_producer_test.go:14]: It's just an exploratory test, It's used to execute code and debug it, check the current behaviour,... etc..

File: synchronizer/l1_worker_test.go (hunk 3 above)
Comment (joanestebanr) [reply synchronizer/l1_worker_test.go:13]: It's just exploratory, to be able to execute real request using etherman to check the results

**Review by ARR552 — APPROVED** [review]

**Pull request merged**